doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -2286,7 +2286,12 @@ def terminal_issue(msg: str) -> List[AssemblyIssue]:
         if major_minor != runtime.get_minor_version():
             return terminal_issue(f"Specified nightly {nightly} does not match group major.minor")
 
-        release_suffix = f"{'konflux-' if runtime.build_system == 'konflux' else ''}release"
+        # For 4.20, remove the -konflux suffix as Konflux builds are being mirrored to standard imagestreams
+        if runtime.build_system == 'brew' or major_minor == '4.20':
```

Comment:
I think there's a constant list available to use instead of the hardcoded value?